src/Exception/StaleFetchException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use RuntimeException;
+
+class StaleFetchException extends RuntimeException implements ShouldRetryException 
+{
+}
```

Comment:
You're probably aware of this, but this will need a `Reference` property, as will `StaleUpdateException`.